openedx/core/djangoapps/models/course_details.py

```diff
@@ -43,6 +47,10 @@ def __init__(self, org, course_id, run):
         self.enrollment_start = None
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
+        self.title = ""
+        self.subtitle = ""
+        self.duration = None
```

Comment:
Duration should be a string, so we should initialize this with an empty string instead of None.